open-craft/ansible-playbooks
# Add ELK roles and playbook

Test Instructions:

1. Go to our ELK server at https://elk.net.opencraft.hosting. Use creds found in https://github.com/open-craft/ansible-secrets/pull/62 to log in (look for kibana basic auth username/pass).
2. Look around to see that logs are available from all of our service infrastructure servers, e.g. MongoDB/MySQL/Postgres/RabbitMQ/Elasticsearch, etc.
3. Try to do some filtering with queries that we were able to do in logz.io.
4. Optionally run the ELK playbook from ansible-secrets *while you have the proper `group_vars` and `deploy` submodules checked out (the `uman/elk` branch). See that it succeeds without issues.
